models/containerized_model/template.py

```diff
@@ -21,7 +21,46 @@ def create_directories(directory_list: list):
             except FileExistsError:
                 pass
 
-    def generate_python():
+    def generate_base_file_metadata():
+        image_name, _ = args.target_docker_image.split(':')
+        deployment_name = image_name.split('/')[-1].strip()
```

Comment:
One think I hit. This name can only be alphanum or "-". I had an image name with "_" and the deployment fails. Maybe we should make deployment name a param with a default. Or make generating the deployment a separate generate action where you give it a name. The latter would mean we could deal with case where there's a generic image (e.g. for scikit models, or for h2o models) but multiple deployments.

Reply:
Have made those changes now. I'm not sure where to consolidate this information in the README, will leave that decision to you.